### DISCUSSION TITLE
Feature Request: Tags

### DISCUSSION THREAD

**Discussion by TheTaylorLee** [linuxserver/Heimdall#985] [Ideas]
I think it would be useful if different tags could be linked from other tag dashboards. This would save a click. Could be implemeted as a pin on other dashboards or possibly even as a drop-down list.

Scenario: You have a work and a personal tag. When you click on the work tag you then have the ability from that dashboard to click on the personal tag and jump directly over to that dashboard.

**Comment by keriati** [comment]
This is already implemented.
